lib/omnibus/packagers/msi.rb

```diff
@@ -88,6 +91,38 @@ class Packager::MSI < Packager::Base
         if signing_identity
           sign_package(msi_file)
         end
+
+        # This assumes, rightly or wrongly, that any installers we want to bundle
+        # into our installer will be downloaded by omnibus and put in the cache dir
+
+        if bundle_msi
+          shellout! <<-EOH.split.join(' ').squeeze(' ').strip
+          candle.exe
+            -nologo
+            #{wix_candle_flags}
+            #{wix_extension_switches(wix_candle_extensions)}
+            -dOmnibusCacheDir="#{windows_safe_path(File.expand_path(Config.cache_dir))}"
+            "#{windows_safe_path(staging_dir, 'bundle.wxs')}"
+          EOH
+
+          bundle_file = windows_safe_path(Config.package_dir, bundle_name)
+
+          bundle_light_command = <<-EOH.split.join(' ').squeeze(' ').strip
+          light.exe
+            -nologo
+            -ext WixUIExtension
+            #{wix_extension_switches(wix_light_extensions)}
+            -cultures:en-us
+            -loc "#{windows_safe_path(staging_dir, 'localization-en-us.wxl')}"
+            project-files.wixobj bundle.wixobj
```

Comment:
Would a bundle need the set of all project-files again?  I would have thought that the chaining process would only need some kind of reference to the msi.